### ISSUE TITLE
An plans for functions to handle arithmetic on image data

### ISSUE THREAD

**Issue by dokeeffe** [nom-tam-fits/nom-tam-fits#71]
Just a query. I'm wondering if there are any plans for features like subtraction of image data, division, median-combine. Things like that. It would be useful for image calibration using darks and flats.
Thanks,
Derek

**Comment by ritchieGitHub** [comment]
Yes, that would be a nice extention. Do you have access to descriptions of those algorithms? Than I would say create an issue for every operation type with a description of the algorithm. Especially in context of fits spezialities like base/scale/blank/minimum/maximum and rounding.

**Comment by dokeeffe** [comment]
Great thanks, I'll have to do some investigation myself on the algorithms.
